### ISSUE TITLE
Error with docker compose 

### ISSUE THREAD

**Issue by amira-essawy** [KIT-MRT/latent_diffusion_face_anonymization#1]
Thank you for your great work.

I am trying to run `docker compose up` but I got this error, can you help with that?
![Screenshot 2024-03-12 165914](https://github.com/KIT-MRT/latent_diffusion_face_anonymization/assets/118816526/e58ef267-1c69-461b-966e-bfeb2d7ff92f)

**Comment by keroe** [comment]
Yeah sure, we had the same error and patched it locally in a feature branch. I merged it to main. Please try out the new version and tell me if it works. It did for me
